src/pages/videos/videos.css

```diff
@@ -0,0 +1,51 @@
+.videoListing-wrapper{
+    margin: 0rem 1rem;
+    grid-area: main;
+}
+
+.videoListing-title{
+    text-align: center;
+}
+
+.videoListing-card-parent{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: stretch;
+    margin: 1rem;
+
+}
+
+.videoListing-card{
+    max-width: 20rem;
+    height: 100%;
+    margin: 1rem;
+    padding: 0.5rem;
+    border: 1px solid;
+    border-radius: 0.2rem;
+}
+
+.videoListing-card-image{
+        object-fit: cover;
+        /* height: 100%; */
+        width: 20rem;
+}
+
+.videoListing-card:hover{
+    box-shadow: 1px 2px 5px 0px #BEC0D8;
```

Comment:
you can create variable for this color to maintain consistency.

Reply:
This is being used only once. SO I think it's fine. I'll take care of it in the next iteration.